examples/mnist_tfaugment2d.py

```diff
@@ -0,0 +1,143 @@
+'''Trains a simple convnet on the MNIST dataset.
```

Comment:
A big problem here is the choice of MNIST to demo data augmentation. Data augmentation is useless on MNIST because the digits are already scaled, centered and oriented. Use CIFAR10 instead.